setup.py

```diff
@@ -0,0 +1,52 @@
+from setuptools import setup, find_packages
+from os import path
+
+
+def _read_files_from_root_dir(*file_basenames: [str]) -> str:
+    """Open and read *file_basenames* from project root dir."""
+
+    return '\n'.join(
+        [open(path.join(path.dirname(__file__), name), encoding='utf8').read()
+         for name in file_basenames]
+    )
+
+
+setup(
+    name='qsck',
+    version='0.0',
+    packages=find_packages('qsck'),
+    url='https://github.com/mblomdahl/qsck',
+    license='The Unlicense',
+    author='Mats Blomdahl',
+    author_email='mats.blomdahl@gmail.com',
+    description='The shitty ".qs" file (de-)serializer',
+    long_description=_read_files_from_root_dir('README.md'),
+    entry_points={
+        'console_scripts': [
+            'qs-parse = qsck.parse_cli:run',
+            'qs-format = qsck.format_cli:run'
+        ]
+    },
+    setup_requires=[
+        'pytest-runner'
```

Comment:
See this page for background: https://docs.pytest.org/en/latest/goodpractices.html#good-integration-practices